client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -9,6 +9,7 @@ define([
 
       $scope.meta = meta;
       $scope.types = angular.copy(CONFIG.GRAPH_TYPES);
+      $scope.optimizationStatus = false;
```

Comment:
I guess the text for RUNNING and NOT_RUNNING should be changed ;) , but overall, it seems a better solution!